code/sum.py

```diff
@@ -108,20 +109,42 @@ def download_video(url):
     video.download(os.getcwd())
     return True
 
+def download_subs(subs):
+    ''' Downloads specified Youtube video's subtitles as a vtt/srt file.
+    args:
+        subs(str): Full url for youtube video
+    returns:
+        True
+    '''
+    ydl_opts = {
+            'outtmpl': '%(title)s.%(ext)s',
+            'subtitlesformat':'srt',
+            'writeautomaticsub':True,
+            'skip_download':True
+            }
+
+    with youtube_dl.YoutubeDL(ydl_opts) as ydl:
+        ydl.download([subs])
```

Comment:
What will be the filename of the subtitle file?

Ideally, I should be consistent such as `1.srt`

Reply:
@AdiChat I've changed the filename to 1.srt, as requested.